absl/functional/internal/any_invocable.h

```diff
@@ -197,7 +197,8 @@ union TypeErasedState {
 template <class T>
 T& ObjectInLocalStorage(TypeErasedState* const state) {
   // We launder here because the storage may be reused with the same type.
-#if ABSL_INTERNAL_CPLUSPLUS_LANG >= 201703L
+#if ABSL_INTERNAL_CPLUSPLUS_LANG >= 201703L && \
```

Comment:
The proposed patch works in my testing, but I think I more principled thing to do would be to use `#if __cpp_lib_launder >= 201606L`. This seems to produce the same result when I try it on godbolt.

https://godbolt.org/z/sGezba1b9